Game.h

```diff
@@ -0,0 +1,46 @@
+#include<iostream>
+#include "Frame.h"
+#include "Snake.h"
+#include "Food.h"
+
+class Game{
+private:
+	int height = 20 ;
+	int width = 20 ;
+	char ** screen;
+	bool gameover = false;
+	int score = 0;
+public:
+	Game(){
+		screen = new char * [height];
```

Comment:
where are you freeing this memory allocated?

Reply:
I am planning to do it in destructor of class Game. but will check if it allows me to because I need added code for input and logic of game. I don't want to free it before necessary.  Will definitely write the code to free it, but a bit latter.